primitives/derive/src/numeric.rs

```diff
@@ -0,0 +1,17 @@
+use core::str::FromStr;
+
+use manyhow::{error_message, Result};
+use proc_macro2::TokenStream;
+use quote::quote;
+
+pub fn numeric_impl(input: TokenStream) -> Result<TokenStream> {
+    let input = input.to_string();
+    let numeric = ::iroha_numeric::Numeric::from_str(&input)
```

Comment:
Does this work with negative numbers? `TokenString::to_string` loves putting extra spaces between tokens and `rust_decimal` had to [work around this](https://docs.rs/rust_decimal_macros/latest/src/rust_decimal_macros/lib.rs.html#57-86)

Reply:
Ah, I see now, iroha's `Numeric` just doesn't support negative numbers :smile: